src/styles/index.scss

```diff
@@ -1,3 +1,60 @@
+@import 'variables';
+
 body {
   margin: 0;
+  height: 100vh;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  color: $black-color;
```

Comment:
The variable `$black-color` should be defined in the `variables.scss` file. Ensure that it is correctly imported and defined there.